llvm/test/tools/llvm-mca/AArch64/Neoverse/V1-forwarding.s

```diff
@@ -911,7 +911,7 @@ bfmlalb z0.s, z0.h, z1.h
 # CHECK-NEXT: [1,0]     D============eeeeeER.    ..   mul	z0.d, p0/m, z0.d, z0.d
 # CHECK-NEXT: [1,1]     D=================eeeER  ..   sdot	z0.s, z1.b, z2.b
 # CHECK-NEXT: [1,2]     D==================eeeER ..   sdot	z0.s, z1.b, z2.b
-# CHECK-NEXT: [1,3]     D=====================eeeER   sdot	z0.s, z0.b, z1.b
+# CHECK-NEXT: [1,3]     .D====================eeeER   sdot	z0.s, z0.b, z1.b
```

Comment:
I don't see why there would be a need for a cycle stall here(and for the other cycle stalls added in this test). Could you explain this change?